content/hi/docs/contribute/_index.md

```diff
@@ -0,0 +1,177 @@
+---
+विषय सूची : संकल्पना 
+शीर्षक : कुबेरनेटेस डाक्यमेन्ट मे योगदान दे 
+linktitle: योगदान 
+मुख्य मेनू : सच 
+no_list: सही 
+weight: 80
+card:
+  नाम : योगदान 
+  weight: 10
+  शीर्षक : कुबेरनेटेस मे योगदान देना शुरू करे 
+---
+
+<!-- अवलोकन  -->
+
+*कुबेरनेटेस सभी योगदानकर्ताओ से सुधार का स्वागत करता है चाहे वो नए हो या अनुभवी   *
+
+{{< ध्यान दे  >}}
+कुबेरनेटेस मे योगदान करने के बारे मे ज्यादा जानकारी के लिए 
+[योगदानकर्ता प्रलेखन ](https://www.kubernetes.dev/docs/) देखे । .
+{{< /ध्यान दे  >}}
+
+ये वेबसाईट  [कुबेरनेटेस सिग डॉक्स ](/docs/contribute/#get-involved-with-sig-docs) द्वारा  देख रेख की जाती है ।
+
+कुबेरनेटेस  प्रलेखन  योगदानकर्ता :
+
+- पुराने विषय को सुधारते है 
+- नए विषय बनाते है 
+- प्रलेखन को दूसरी भाषा मे बदलते है 
+- कुबेरनेटेस रिलीज चक्र मे प्रलेखन की देख रेख और प्रकाशन करते है 
+
+
+
+<!-- body -->
+
+## शुरू करना 
+
+कोई भी प्रलेखन के बारे मे मुद्दा खोल सकते है  या कुबेरनेटेस वेबसाईट 
+[`kubernetes/website` GitHub repository](https://github.com/kubernetes/website)मे बदलाव  का योगदान पुल रीक्वेस्ट (PR) द्वारा कर सकते है । 
+आपको 
+[गिट ](https://git-scm.com/) और 
+[गीठब ](https://lab.github.com/) 
```

Comment:
The spaces here before `]` aren't needed.

Reply:
I don't think Git and GitHub should be transliterated because they are technologies/tools that don't have an official transliterated term.